src/child/components/Favourite.js

```diff
@@ -76,11 +91,33 @@ class Favourite extends Component {
         }
     }
 
+    onModalClick(e) {
+        e.stopPropagation();
+    }
+
+    onBackdropClick(e) {
+        this.setState({ isHovered: false });
+        this.props.bindings.onBackdropClick(e);
+    }
+
+    flipModal(starTopPos) {
```

Comment:
I'm probably just being slow, but it took me a while to work out what this means... does `shouldPositionAboveStar` make sense (and I think it's more descriptive than `flipmodal`)?